spec/orangelight/apostrophe_spec.rb

```diff
@@ -112,7 +112,7 @@
 
   describe 'in left_anchor search' do
     let(:params) do
-      { 'q' => "{!qf=$left_anchor_qf pf=$left_anchor_pf}can’t*" }
+      { qf: "${left_anchor_qf}", pf: "${left_anchor_pf}", q: "can't*" }
```

Comment:
Was there a reason to change from the fancy quote to the standard quote?

Reply:
We just didn't notice! Thanks!